gui-agent/vmside.c

```diff
@@ -2240,6 +2256,12 @@ int main(int argc, char **argv)
 
     parse_args(&g, argc, argv);
 
+    /* Clipboard wipe functionality is controlled by the
+     * qubes-gui-agent-clipboard-wipe service
+     */
+    g.clipboard_wipe =
+        access("/run/qubes-service/qubes-gui-agent-clipboard-wipe", F_OK) == 0;
```

Comment:
It would be nice to announce support for this using https://dev.qubes-os.org/projects/core-admin/en/latest/qubes-features.html#services. But then, there is a catch: qubesdb keys are limited to 63 characters, so `/features-request/supported-service.<service-name>` needs to fit there, and with the current name it does not. Maybe drop `qubes-` prefix?

BTW, announcing in this repo is to be added in `appvm-scripts/etc/qubes/post-install.d/20-qubes-guivm-gui-agent.sh`